dataprofiler/tests/profilers/test_unstructured_text_profile.py

```diff
@@ -237,21 +237,21 @@ def test_options_stop_words(self):
         text_profile.update(sample)
 
         expected_word_count = {'This': 1, 'Test': 1, 'test': 1}
-        expected_vocab = [' ', ',', '.', '!', 'T', 'h', 'i', 's', 'a',
-                          'e', 't', 'n', 'c']
+        expected_vocab = {'s': 5, ' ': 5, 'e': 5, 't': 4, '!': 3, 'T': 2,
+                          'i': 2, 'n': 2, 'h': 1, ',': 1, 'a': 1, 'c': 1, '.': 1}
         self.assertDictEqual(expected_word_count, text_profile.word_count)
-        self.assertCountEqual(expected_vocab, text_profile.vocab)
+        self.assertDictEqual(expected_vocab, text_profile.vocab_count)
 
         ## input with two samples
         text_profile = TextProfiler("Name", options=options)
         sample = pd.Series(["This is test,", " a Test sentence.!!!"])
         text_profile.update(sample)
 
         expected_word_count = {'This': 1, 'Test': 1, 'test': 1}
-        expected_vocab = [' ', ',', '.', '!', 'T', 'h', 'i', 's', 'a',
-                          'e', 't', 'n', 'c']
+        expected_vocab = {'s': 5, ' ': 5, 'e': 5, 't': 4, '!': 3, 'T': 2,
+                          'i': 2, 'n': 2, 'h': 1, ',': 1, 'a': 1, 'c': 1, '.': 1}
         self.assertDictEqual(expected_word_count, text_profile.word_count)
-        self.assertCountEqual(expected_vocab, text_profile.vocab)
+        self.assertDictEqual(expected_vocab, text_profile.vocab_count)
```

Comment:
should we test both?

Reply:
same for above

Reply:
after the change, the text profile doesn't have attribute `vocab` (it's consistent with the word), so I removed that test